### PR TITLE
progress bar for cards

### PR DESCRIPTION
**Pull request by trilopin**
## Summary
Add progress bar to run cards:
- added computation at playbookrun resolver
- add new field in graphql query
- use field to draw green line at run card

Screenshot 2022-12-02 at 13 41 42

## Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-playbooks/issues/1604

## Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

**Review by gbochora — APPROVED** [review]
Cool!

**Review by lieut-data — APPROVED** [review]
Sweet! This is the first GraphQL usage where it "clicks" for me, exposing a computationally derived field that we can choose to send down when requested. Super cool :)

Two comments.

**Comment by trilopin** [comment]
> Sweet! This is the first GraphQL usage where it "clicks" for me, exposing a computationally derived field that we can choose to send down when requested. Super cool :)
> 

Imagine how easily could you replace some complex SQL blocks like the one for NumActions.

File: webapp/src/components/rhs/rhs_run_list.tsx [hunk 1]
```diff
@@ -360,38 +361,53 @@ const RHSRunListCard = (props: RHSRunListCardProps) => {
...
+    padding:0;
+    border-radius: 4px;
+    &:after {
+        border-bottom: 2px solid var(--online-indicator);
+        content: '';
+        display: block;
+        // sad hack to make the progress bar work with border-radius
+        margin: 0 5px;
+        width: ${({progress}) => (progress > 98 ? 98 : progress)}%;
+    }
```
[the remark this answers is not in the document]
Comment (trilopin) [reply webapp/src/components/rhs/rhs_run_list.tsx]: Thanks @lieut-data ❤️

**Pull request merged**